project.clj

```diff
@@ -25,11 +25,12 @@
                      :url                     "http://www.apache.org/licenses/LICENSE-2.0"}
   :min-lein-version "2.8.1"
   :repositories     [["sonatype-snapshots" {:url "https://oss.sonatype.org/content/groups/public" :snapshots true}]
-                     ["jitpack"            {:url "https://jitpack.io"}]]
+                    ["jitpack" "https://jitpack.io"]]
   :plugins          [
                       [org.noisesmith/git-info-edn "0.2.1"]
                     ]
   :dependencies     [
+                      [com.github.maoo.symphony-java-client/symphony-client "mbeans-rc-1" :exclusions [org.slf4j/slf4j-log4j12]]
```

Comment:
When do you expect SJC to merge your changes upstream?  I have a strong preference for waiting for that to happen, rather than having the bot add dependencies to a fork.

Reply:
They are already merged to dev branch and deployed to `symphony-client-1.1.3-SNAPSHOT` , see https://travis-ci.org/symphonyoss/symphony-java-client . I will test with the SNAPSHOT version and keep you posted.